.vscode/settings.json

```diff
@@ -0,0 +1,8 @@
+{
+  "editor.formatOnSave": true,
```

Comment:
format on save: false, bc eslint does the formatting for you using the eslint-config-prettier and plugin